libbeat/kibana/fields_transformer_test.go

```diff
@@ -99,14 +100,59 @@ func TestMissingVersion(t *testing.T) {
 func TestDuplicateField(t *testing.T) {
 	commonFields := common.Fields{
 		common.Field{Name: "context", Path: "something"},
-		common.Field{Name: "context", Path: "something", Type: "keyword"},
+		common.Field{Name: "context", Path: "something", Type: "date"},
```

Comment:
The problem with this change is that so far it also told us when there were duplicate identical definitions. I assume now it will only tells us if there are conflicting definitions?

Reply:
Right.  One idea for combining those options - log a warning unless `overwrite` is set?  The benefit would be an easier transition for users that are using ECS fields already - I'm not sure how much of a benefit that is.

@ruflin I'm happy either way - please let me know if you'd prefer a failure or a warning log if `overwrite` is not set.

Reply:
Implemented failure for now.

Reply:
Sorry for the late answer. I also prefer failure over warning as warning is not very actionable. I rather break the build if something is not done properly.